app/gui/src/components/TabView.vue

```diff
@@ -136,6 +152,7 @@ const onSignOut = () => {
 .tablist {
   display: flex;
   flex-direction: row;
+  z-index: 0;
```

Comment:
Is this to create a stacking context? A comment about what problem this prevents would be useful

Reply:
For stacking context it's better to use 'isolation: isolate'